sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Release History
 
-## 1.15.1 (Unreleased)
+## 1.16.0 (Unreleased)
+
+### New Features
+
+- Add new ***provisional*** methods `send_request` onto the `azure.core.PipelineClient` and `azure.core.AsyncPipelineClient`. This method takes in
+requests and sends them through our pipelines.
+- Add new ***provisional*** module `azure.core.rest`. `azure.core.rest` is our new public simple HTTP library in `azure.core` that users will use to create requests, and consume responses. Can only be used with the provisional method `send_request` on our `PipelineClient`s
```

Comment:
get rid of `PipelineClient` `send_request` mention in `azure.core.rest` changelog entry.

Instead, in doc for azure.core.rest, tell them to pass it to their client's `send_request`